COMMANDS.org

```diff
@@ -0,0 +1,117 @@
+#+TITLE: Nix Command Reference
+#+AUTHOR: github.com/Pitta
+
+A reference to ~nix~ commands found to be helpful in day to day use.
+
+* Package Management
+
+Coming from package managers like ~brew~, ~port~, ~apt~, etc., ~nix~ can /seem/ daunting/impenetrable. These section should help demystify how to use ~nix~ as a drop in replacement for how most folks approach using the aforementioned tools.
+
+- [[#-searching-for-packages][Searching for Packages]]
+- [[#-imperitive-package-management][Imperitive Package Management]]
+  - how most users manage packages, typically by invoking an install command with the package manager of choice.
+- [[#-declaritive-package-management][Declaritive Package Management]]
+  - A more portable, version controll-able approach that declares the desired end state. Great for using the same configuration across multiple machines.
+- [[#-temporary-packages][Temporary Packages]]
+  - Excellent for testing new versions of packages, or entirely new tools, without comitting to system level changes that could impact other parts of the system.
```

Comment:
I wouldn't include these at all. You already get a free table of contents because you're using headers. The sub bullets make more sense as the first line of each section. It made me think they were linking out to other websites instead of sections